docker-compose.production.yaml

```diff
@@ -4,8 +4,13 @@ services:
 
   app:
     image: usedgoodreads/app:production
-    ports:
-      - "80:5000"
 
   worker:
     image: usedgoodreads/app:production
+
+  nginx:
+    volumes:
+      - /etc/letsencrypt/live/usedgoodreads.com/:/certs
+    ports:
+      - 80:80
+      - 443:443
```

Comment:
Make it strings please

https://docs.docker.com/compose/compose-file/#ports

> When mapping ports in the HOST:CONTAINER format, you may experience erroneous results when using a container port lower than 60, because YAML parses numbers in the format xx:yy as a base-60 value. For this reason, we recommend always explicitly specifying your port mappings as strings.

Reply:
Fixed in cedc53afb99cdda9d67c86db6ac8f7ebb2c7c887